src/OrchardCore.Modules/OrchardCore.Users/Endpoints/EmailAuthenticator/SendCode.cs

```diff
@@ -0,0 +1,116 @@
+using System.Collections.Generic;
+using System.IO;
+using System.Text.Encodings.Web;
+using System.Threading.Tasks;
+using Fluid.Values;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.Extensions.Localization;
+using OrchardCore.Email;
+using OrchardCore.Liquid;
+using OrchardCore.Settings;
+using OrchardCore.Users.Models;
+
+namespace OrchardCore.Users.Endpoints.EmailAuthenticator;
+
+public static class SendCode
+{
+    public const string RouteName = "EmailAuthenticatorSendCode";
```

Comment:
No need because the route will be registered once

Reply:
It's needed because it's used by other places like the view

Reply:
I know what are you doing, no problem but we could strong type all the routes in other modules if you want to follow what you are started